common/utils/src/main/scala/org/apache/spark/util/SparkStringUtils.scala

```diff
@@ -56,6 +56,9 @@ private[spark] trait SparkStringUtils {
     import org.apache.spark.util.ArrayImplicits._
     str.split(",").map(_.trim()).filter(_.nonEmpty).toImmutableArraySeq
   }
+
+  /** Try to strip prefix and suffix with the given string 's' */
+  def strip(str: String, s: String): String = str.stripPrefix(s).stripSuffix(s)
```

Comment:
StringUtils.strip can handle null properly, I think that is only difference here. Since it is used to handle strings in `sys.env("PATH")`, looks like there won't be null. But if this will be used in other places, maybe null handle is good to add.

Reply:
i.e., if `str` is null, this will be NPE.